src/reverselock.h

```diff
@@ -13,7 +13,7 @@ class reverse_lock
 {
 public:
 
-    explicit reverse_lock(Lock& lock) : lock(lock) {
+    explicit reverse_lock(Lock& _lock) : lock(_lock) {
```

Comment:
Well in the case of the GUI that was the decision, as an exception.

As I said there, for the consensus code I would require the binaries to be the same. This is much closer to that.